tests/integration/test_api/test_statistics/test_statistics_format.py

```diff
@@ -0,0 +1,214 @@
+t2_cases_path = os.path.join(TEST_CASES_PATH, 'agent_statistics_format_test_module',
+                             'cases_agent_statistics_format.yaml')
+t2_statistics_template_path = os.path.join(STATISTICS_TEMPLATE_PATH, 'agent_statistics_format_test_module')
+
+# Agent statistics format test configurations (t2)
+test2_configuration, test2_metadata, test2_cases_ids = get_test_cases_data(t2_cases_path)
+
+
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('test_configuration,test_metadata', zip(test1_configuration, test1_metadata),
+                         ids=test1_cases_ids)
+def test_manager_statistics_format(test_configuration, test_metadata, load_wazuh_basic_configuration,
+                                   set_wazuh_configuration, daemons_handler):
+    """
+    description: Check if the statistics returned by the API have the expected format.
+
+    test_phases:
+        - setup:
+            - Load Wazuh light configuration
+            - Apply ossec.conf configuration changes according to the configuration template and use case
+            - Restart wazuh-manager service to apply configuration changes
+        - test:
+            - Request the statistics of a particular daemon from the API
+            - Compare the obtained statistics with the json schema
+        - tierdown:
+            - Restore initial configuration
+            - Stop wazuh-manager
+
+    wazuh_min_version: 4.4.0
+
+    parameters:
+        - test_configuration:
+            type: dict
+            brief: Configuration data from the test case.
+        - test_metadata:
+            type: dict
+            brief: Metadata from the test case.
+        - load_wazuh_basic_configuration:
+            type: fixture
+            brief: Load basic wazuh configuration.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Apply changes to the ossec.conf configuration.
+        - daemons_handler:
+            type: fixture
+            brief: Wrapper of a helper function to handle Wazuh daemons.
+
+    assertions:
+        - Check if the statistics returned by the API have the expected format.
+
+    input_description:
+        - The `configuration_manager_statistics_format` file provides the module configuration for this test.
+        - The `cases_manager_statistics_format` file provides the test cases.
+    """
+    endpoint = test_metadata['endpoint']
+    statistics_schema_path = os.path.join(t1_statistics_template_path, f"{endpoint}_template.json")
+    params = f"?daemons_list={endpoint}"
+    url = get_base_url() + DAEMONS_STATS_ROUTE + params
+    authentication_headers, _ = login()
+
+    # Get daemon statistics
+    response = requests.get(url, headers=authentication_headers, verify=False)
+    stats_schema = read_json_file(statistics_schema_path)
+
+    # Check if the API statistics response data meets the expected schema. Raise an exception if not.
+    jsonschema.validate(instance=response.json(), schema=stats_schema)
```

Comment:
Do this in the framework. Check for example: https://github.com/wazuh/qa-integration-framework/blob/17441-api-tests-migration/src/wazuh_testing/modules/analysisd/utils.py#L24

Reply:
Done